test/remapping.jl

```diff
@@ -72,149 +72,291 @@ end
+            m2, n2 = 2, 2
+            mesh2 = Meshes.EquispacedRectangleMesh(domain2, m2, n2)
+            target_topo = Topologies.GridTopology(mesh2)
+            target = Spaces.SpectralElementSpace2D(target_topo, quad)
+
+            R = LinearRemap(target, source)
+            R_true = I
+
+            @test R.map ≈ R_true
+            @test nnz(R.map) == 4
+
+            @test !conservative(R)
+            @test consistent(R)
+            @test monotone(R)
+
+            @testset "Scalar Remap Operator Application" begin
+                n = length(source.local_geometry)
+                source_field =
+                    Fields.Field(IJFH{FT, 1}(ones(1, 1, n, 1)), source)
+
+                # test consistent remap
+                target_field = remap(R, source_field)
+                @test vec(parent(target_field)) ≈
+                      ones(length(target.local_geometry))
+
+                # test simple remap
+                vec(parent(source_field)) .= [1.0; 2.0; 3.0; 4.0]
+                remap!(target_field, R, source_field)
+                @test vec(parent(target_field)) ≈ [1.0; 2.0; 3.0; 4.0]
+            end
+        end
     end
 
-    @testset "Concentric Domains of Different Area" begin
-        domain1 = Domains.RectangleDomain(
-            Geometry.XPoint(-1.0)..Geometry.XPoint(1.0),
-            Geometry.YPoint(-1.0)..Geometry.YPoint(1.0),
-            x1boundary = (:bottom, :top),
-            x2boundary = (:left, :right),
-        )
-
-        m1, n1 = 2, 2
-        mesh1 = Meshes.EquispacedRectangleMesh(domain1, m1, n1)
-        source_topo = Topologies.GridTopology(mesh1)
-        source = Spaces.SpectralElementSpace2D(source_topo, quad)
-
-        domain2 = Domains.RectangleDomain(
-            Geometry.XPoint(-.5)..Geometry.XPoint(0.5),
-            Geometry.YPoint(-.5)..Geometry.YPoint(0.5),
-            x1periodic = true,
-            x2periodic = true,
-        )
-        m2, n2 = 2, 2
-        mesh2 = Meshes.EquispacedRectangleMesh(domain2, m2, n2)
-        target_topo = Topologies.GridTopology(mesh2)
-        target = Spaces.SpectralElementSpace2D(target_topo, quad)
-
-        R = LinearRemap(target, source)
-        R_true = I
-
-        @test R.map ≈ R_true
-        @test nnz(R.map) == 4
-
-        @test !conservative(R)
-        @test consistent(R)
-        @test monotone(R)
-
-        @testset "Scalar Remap Operator Application" begin
-            n = length(source.local_geometry)
-            source_field = Fields.Field(IJFH{FT, 1}(ones(1, 1, n, 1)), source)
-
-            # test consistent remap
-            target_field = remap(R, source_field)
-            @test vec(parent(target_field)) ≈
-                  ones(length(target.local_geometry))
-
-            # test simple remap
-            vec(parent(source_field)) .= [1.0; 2.0; 3.0; 4.0]
-            remap!(target_field, R, source_field)
-            @test vec(parent(target_field)) ≈ [1.0; 2.0; 3.0; 4.0]
+    @testset "1D Domains" begin
```

Comment:
Besides my little OCD, I think it's more natural and organized to put the 1D case (both in tests and in the API functions) first and then the higher-dim cases